uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/TokenRequestTimeout.java

```diff
@@ -29,7 +29,7 @@
  */
 public enum TokenRequestTimeout {
 
-    SILENT(15, TimeUnit.SECONDS),
+    SILENT(20, TimeUnit.SECONDS),
```

Comment:
Do we know which specific silent requests are taking such long time? I was actually hoping that even 15 seconds was way more than enough

Reply:
I think this was a leftover attempted fix for something, so i'm going to get rid of this change